src/main/java/lotto/domain/LottoMoney.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+public class LottoMoney {
+    private final String MONEY_MUST_OVER_1000 = "구매 금액은 1000원 이상을 입력해주세요.";
+    private final String MONEY_UNIT_IS_1000 = "구매 금액은 1000원 단위로 입력해주세요.";
+    private final int LOTTO_PRICE_UNIT = 1000;
```

Comment:
```suggestion
    private static final String MONEY_MUST_OVER_1000 = "구매 금액은 1000원 이상을 입력해주세요.";
    private static final String MONEY_UNIT_IS_1000 = "구매 금액은 1000원 단위로 입력해주세요.";
    private static final int LOTTO_PRICE_UNIT = 1000;
```
상수 선언은 보통 static final을 함께 사용합니다!